### PR TITLE
⬆️ Terraform and Azure CLI Version update, use non-root user

### PR DESCRIPTION
**Pull request by bgauduch**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -41,16 +41,23 @@ LABEL maintainer="bgauduch@github"
...
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
-    ca-certificates=20190110 \
-    git=1:2.20.1-2+deb10u3 \
-    python3=${PYTHON_MAJOR_VERSION}.3-1 \
-    python3-distutils=${PYTHON_MAJOR_VERSION}.3-1 \
+  ca-certificates=20190110 \
+  git=1:2.20.1-2+deb10u3 \
+  python3=${PYTHON_MAJOR_VERSION}.3-1 \
+  python3-distutils=${PYTHON_MAJOR_VERSION}.3-1 \
```
Comment (pyaillet) [inline Dockerfile:47]: Why did you remove the indentation for the package list ?

**Review by pyaillet — DISMISSED** [review]

File: Dockerfile (hunk 1 above)
Comment (bgauduch) [reply Dockerfile:47]: Good question, certainly a linter issue on my side !
Will revert that back to normal

**Review by pyaillet — APPROVED** [review]
LGTM
